components/SponsorsSection.js

```diff
@@ -62,6 +62,8 @@ function ItemSponsor ({ name, src, type }) {
       <style jsx>{`
         .sponsor-item-gold, .sponsor-item-silver {
           display: inline-block;
+          cursor: pointer;
```

Comment:
This does not make each sponsor a link. Please update your branch against latest `master` branch in this repository.